QueryCiv3/BiqSections/Tile.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct TILE
+    {
+        // Not 100% sure the flags are correct. The Apolyton documentation gets confusing here
+        public int Length;
+
+        private byte RiverConnections;
+        public bool RiverConnectionNorth { get => Util.GetFlag(RiverConnections, 0); }
+        public bool RiverConnectionWest  { get => Util.GetFlag(RiverConnections, 1); }
+        public bool RiverConnectionEast  { get => Util.GetFlag(RiverConnections, 2); }
+        public bool RiverConnectionSouth { get => Util.GetFlag(RiverConnections, 3); }
+
+        public byte Border;
+        public int Resource;
+        public byte TextureLocation; // Between 0 and 80 inclusive (9*9 grid)
+        public byte TextureFile; // 0: xtgc, 1: xgpc, 2: xdgc, 3: xdpc, 4: xdgp, 5: xggc, 6: wcso, 7: wsss, 8: wooo
+        private fixed byte UnknownBuffer[2];
+        public byte OverlayFlags;
+        public byte Terrain;
+        public byte BonusFlags;
+        public byte RiverCrossingFlags;
```

Comment:
Subject to confirmation/disproval, I have a comment in my editor for this byte:

`//appears to always be zero.  can use river connection info to figure out where rivers are instead`

Since there are at least three places referring to river info, we should probably be verbose and proactive about documenting what we can about them here.